front/app/components/admin/ContentBuilder/Widgets/Title/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+// styling
+import styled, { useTheme } from 'styled-components';
+
+// craft
+import { useNode } from '@craftjs/core';
+
+// components
+import {
+  Box,
+  Input,
+  Title as TitleComponent,
+} from '@citizenlab/cl2-component-library';
+
+// i18n
+import messages from './messages';
+import { useIntl } from 'utils/cl-intl';
+
+const H3 = styled.h3<{ color: string }>`
```

Comment:
Is this a title or a heading. But assuming as we're only providing H3 then there's no flexibility for the user to use this for sub headings also? They can use text for that anyhow.

Reply:
Yeah you could also call it a heading I guess, in the component library it's also called a 'Title'. My idea was to allow the user to use different heading with different sizes in the future